pkgs/development/python-modules/mouseinfo/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, pyperclip
+, fetchFromGitHub
+, xlib
+}:
+buildPythonPackage rec {
+  pname = "MouseInfo";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "asweigart";
+    repo = "mouseinfo";
+    rev = "1876ad5cd311b4352d46bc64a12edfb4da49974e";
+    sha256 = "sha256-UTaHTJE0xFihN9r+DY/WhekZ7S/CXtMFbqAayzexRxk=";
+  };
+
+  patches = [
+    ./fix-xlib-version.patch
+  ];
+
+  doCheck = false;
```

Comment:
The project has a test directory.

Reply:
It requires a X server running to work and it's a no-op that practically only imports the module https://github.com/asweigart/mouseinfo/blob/27d4059a20180344bbfaed4b679f193d44c9f366/tests/test_mouseinfo.py#L6